authentik/root/settings.py

```diff
@@ -305,14 +305,15 @@
     }
 }
 
-if CONFIG.get_bool("postgresql.use_pgpool", False):
-    DATABASES["default"]["DISABLE_SERVER_SIDE_CURSORS"] = True
-
-if CONFIG.get_bool("postgresql.use_pgbouncer", False):
-    # https://docs.djangoproject.com/en/4.0/ref/databases/#transaction-pooling-server-side-cursors
-    DATABASES["default"]["DISABLE_SERVER_SIDE_CURSORS"] = True
-    # https://docs.djangoproject.com/en/4.0/ref/databases/#persistent-connections
-    DATABASES["default"]["CONN_MAX_AGE"] = None  # persistent
+# https://docs.djangoproject.com/en/4.0/ref/databases/#transaction-pooling-server-side-cursors
+DATABASES["default"]["DISABLE_SERVER_SIDE_CURSORS"] = CONFIG.get_bool(
+    "postgresql.disable_server_side_cursors", False
+)
+# https://docs.djangoproject.com/en/4.0/ref/databases/#persistent-connections
+# Not a PostgreSQL-specific setting, but a database-specific setting.
+# Only PostgreSQL is supported as a database, so we place this setting it in the 'postgresql' path.
+DATABASES["default"]["CONN_MAX_AGE"] = CONFIG.get("postgresql.conn_max_age", None)
```

Comment:
Ideally, all of that would directly happen above in the `DATABASES["default"]` dict directly.